packages/cxl-lumo-styles/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@conversionxl/cxl-lumo-styles",
-  "version": "1.6.3",
+  "version": "1.6.4",
```

Comment:
I'm not sure it's best practice to bump package versions with each PR. If so, it would be better in a separate commit.

@pawelkmpt @lkraav Thoughts?